src/HomePage.js

```diff
@@ -1,11 +1,12 @@
 import React, { Fragment } from 'react';
+import { DragDropContext} from 'react-beautiful-dnd';
 
 import {ModalCard} from './ModalCard';
 import {CardInfo} from './CardInfo';
 import {Section} from './Section';
 import {CardAddPanel} from './CardAddPanel';
 
-import {findCardById} from './appFunctions';
+import {findCardById, changeStatusCard} from './appFunctions';
 import {cardCreate, cardDelete, cardChange} from './networkFunctions';
```

Comment:
please use one styling for imports: with spaces before/after curly brace or without spaces